make/staticbuild/linux_cpu.mk

```diff
@@ -37,7 +37,7 @@ DEBUG = 0
 USE_SIGNAL_HANDLER = 1
 
 # the additional link flags you want to add
-ADD_LDFLAGS += -L$(DEPS_PATH)/lib -lpng -ltiff -ljpeg -lz -lgfortran -ldl  -Wl,--version-script=$(CURDIR)/make/config/libmxnet.ver,-rpath,'$${ORIGIN}',--gc-sections
```

Comment:
What's the reason for the ssl symbols to be part of the library previously even though they are not listed in libmxnet.ver?

Reply:
It is caused by some other packages looking for openssl functions. If the package contains openssl dependencies, they may end up loading MXNet ones before loading its own. The mismatch on ssl versions caused the other packages malfunctioning.

Reply:
My point is that if `libmxnet.ver` were applied correctly, mismatch on ssl versions will be impossible. Are you sure that your current `libmxnet.so` used the `-Wl,--version-script=$(CURDIR)/make/config/libmxnet.ver`? If so, I'm not sure if this PR will resolve the problem. The `-Wl,--exclude-libs,ALL` is more robust than the version script, but in the end they serve the same purpose.